clarity/evaluator/msbg/msbg_utils.py

```diff
@@ -105,167 +105,185 @@ def read_gtf_file(gtf_file):
     # Fix filename if necessary
     dirname = os.path.dirname(os.path.abspath(__file__))
     gtf_file = os.path.join(dirname, gtf_file)
-    with open(gtf_file, "r", encoding="utf-8") as fp:
-        data = json.load(fp)
+    with open(gtf_file, "r", encoding="utf-8") as file_pointer:
+        data = json.load(file_pointer)
     for key in data:
         if isinstance(data[key], list):
             data[key] = np.array(data[key])
     return data
 
 
-def firwin2(n, f, a, window=None, antisymmetric=None):  # pylint: disable=W0613
+def firwin2(
+    n_taps: int,
+    sample_frequency: np.ndarray,
```

Comment:
I think here, "f" needs to be "frequencies" - it defining a set of frequency positions at which the gains will be specified.  "sampling_frequency" would more likely be understood as the specific frequency at which the signal has been sampled.